roles/principal_data_engineer.md

```diff
@@ -0,0 +1,59 @@
+
+Our Principal Data Engineers enable public sector organisations to transform service delivery with data to improve society, for everyone. They provide our most senior data leadership and support to our clients and teams, build and manage key relationships with clients, manage data consultancy and delivery across one or more accounts, and support sales.
+
+## What does the job entail?
+
+At Made Tech we want to positively impact the future of the country by using technology to improve society, for everyone. We want to empower the public sector to deliver and continuously improve digital services that are user-centric, data-driven and freed from legacy technology. To do this we work alongside brilliant public servants to drive better decisions through data and automation.
+
+As a Principal Data Engineer you will work closely with our customers senior leadership to help inform their data strategy and to ensure our teams are delivering to a high quality and are aligned to our customers data vision. Working alongside delivery, client and other capability principals, you will help shape account plans and be responsible for delivering against them.
+
+Working with Made Tech's leadership you will ensure our work with customers is aligned to our growth goals and you will play a pivotal role in identifying new opportunities and winning work. Running bid teams, supporting sales teams and developing new business is critical to this role.
+
+You will coach and support Lead Data Engineers to steer their team's towards success. You will also be responsible for hiring and line managing Lead Data Engineers.
+
+While this is not a hands-on coding technical role, the importance of credibility in internet-era approaches to digital, data and technology in the public sector cannot be understated. You will be expected to maintain a broad technical knowledge of modern data practices, be able to shape data strategy and roadmaps, and hold others to account for technical quality.
+
+As a data leader within Made Tech you will be expected to maintain and grow your professional network. You will be expected to contribute to thought leadership, content and events and should have a proven track record of doing so.
+
+## What experience are we looking for?
+
+While we will look for you to have experience in these things, if you don’t have one of these don’t let that stop you from applying.
+
+-   Working directly with customers and users    
+-   Working within a technology consultancy
+-   Shaping data strategy and approaches to quality, ethics and governance
+-   Developing a data capability or function
+-   Combining data science, analytics and engineering
+-   Strong understanding of various architectures including data warehouses, data lakes, data lake houses and data mesh
+-   Strong understanding of best practice DataOps and MLOps
+-   Up-to-date understanding of various data engineering technologies including Apache Spark, Databricks and Hadoop
```

Comment:
I worry that from this point in the list, we're actually listing _more_ technical requirements than in our lead and senior roles - reckon we could trim a few here?

Reply:
The principal data engineer was from Jim. I guess the idea here is to have a principal with. enough technical background.